css/properties/transform-origin.json

```diff
@@ -225,10 +225,12 @@
                 "version_added": "14"
               },
               "safari": {
-                "version_added": "6"
+                "version_added": "6",
+                "notes": "Only supported for transformations applied using the CSS `transform:` property. It has no effect on transformations applied using the `transform=\"\"` SVG attribute."
```

Comment:
```suggestion
                "notes": "Only supported for transformations applied using the CSS <code>transform</code> property. It has no effect on transformations applied using the <code>transform</code> SVG attribute."
```

See https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#notes —
> The `<code>`, `<kbd>`, `<em>`, and `<strong>` HTML elements may be used. In addition, `<a>` tags may be used, such as to link to a browser's bug report, or MDN documentation.

And IMHO for consistency with the way CSS property names are marked up in other notes, it’s better to omit the trailing `:` colon character in the CSS property name, and better to omit the `=""` part after the `transform` markup attribute name.

For precedents, see the following:

https://github.com/mdn/browser-compat-data/blob/7bcdd8ff424a8eef2bb24aa79cf08d4803282063/css/properties/text-transform.json#L10

https://github.com/mdn/browser-compat-data/blob/7bcdd8ff424a8eef2bb24aa79cf08d4803282063/html/elements/script.json#L152

Reply:
Ah, yes - thank you, I wasn't sure what the syntax for marking-up code-text was - I assumed Markdown was supported.

I understand your removal of the `:` and `=""`, but I'd like some way for readers to immediately know which `transform` is the CSS property and which is the SVG attribute. Could an example be added, e.g.

```
"notes": "Only supported for transformations applied using the CSS <code>transform</code> property (e.g. <code>.className { transform: rotate(45deg); transform-origin: center; }</code>). It has no effect on transformations applied using the <code>transform</code> SVG attribute (e.g. <code>&lt;rect style="transform-origin: center;" transform="rotate(45)" /&gt;</code> )."
```

Reply:
> I'd like some way for readers to immediately know which `transform` is the CSS property and which is the SVG attribute. Could an example be added

Yep, I think that would be fine